template/build/core/phing/tasks/setup.xml

```diff
@@ -7,12 +7,6 @@
     <echo></echo>
   </target>
 
-  <target name="acsf:init" description="Initializes ACSF support for project.">
```

Comment:
Im assuming this is removed because it has nothing to do with a local environment.